test/benchmarks/Benchmarks.Trace/Benchmarks.Trace.csproj

```diff
@@ -28,6 +28,7 @@
 
   <ItemGroup>
     <PackageReference Include="BenchmarkDotNet" Version="0.12.1" />
+    <PackageReference Include="JetBrains.Profiler.Api" Version="1.1.7" />
```

Comment:
We should add this to `LICENSE-3rdparty.csv`.

Reply:
done in 5e27760

Reply:
I re-opened this because I'm surprised we need to include this in our 3rd party license file, as we're not redistributing their code. Do we really need to add that?

Reply:
No, I think Zach is right. I just had this same conversation with @andrewlock in another PR. Just me being too aggressive about `LICENSE-3rdparty.csv`.